src/virtual-keyboard.directive.ts

```diff
@@ -20,9 +20,10 @@ export class NgVirtualKeyboardDirective {
   private opened = false;
   private focus = true;
 
-  @Input('ng-virtual-keyboard-layout') layout: KeyboardLayout|string;
+  @Input('ng-virtual-keyboard-layout') layout: KeyboardLayout | string;
   @Input('ng-virtual-keyboard-placeholder') placeholder: string;
   @Input('ng-virtual-keyboard-type') type: string;
+  @Output('ng-virtual-keyboard-close') keyboardClose: EventEmitter<void> = new EventEmitter();
```

Comment:
Personally I prefer to initialize these on `construct` method - but making this like this isn't really a problem just personal preference.